mmv1/products/secretmanager/terraform.yaml

```diff
@@ -61,6 +61,11 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         name: secret_data
         sensitive: true
         custom_expand: templates/terraform/custom_expand/base64.go.erb
+      version: !ruby/object:Overrides::Terraform::PropertyOverride        
+        name: "version"
+        output: true
+        input: true
```

Comment:
`output` means that only the API controls the value, and the user cannot modify it. If that is true, then `input` should not be here as well (which states that if the user wants to change this field, it must be destroyed and recreated).